src/scipp/plotting/__init__.py

```diff
@@ -86,6 +86,15 @@ def plot(*args, **kwargs):
         is used. `labels={"time": "time-labels"}`. Defaults to `None`.
     :type labels: dict, optional
 
+    :param camera: Dict configuring the camera. Valid entries are 'position' and
+        'look_at'. This option is valid only for 3-D scatter plots.
+        The 'position' entry defined the position of the camera and the 'look_at'
+        entry defines the point the camera is looking at.
+        Both must be scalar variables with the correct unit, i.e., the unit of the
```

Comment:
Not sure about "scalar variables", since they should in fact be a vector. I know they are still `scalar` with a vector dtype, but "scalar" may be misleading for users?